operator/mapper/dbaas_test.go

```diff
@@ -0,0 +1,70 @@
+package mapper
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_CompareMajorVersion(t *testing.T) {
+	tests := map[string]struct {
+		versionA     string
+		versionB     string
+		expectedBool bool
+		shouldError  bool
```

Comment:
Instead of a bool, I'd propose to check the specific error message.
```suggestion
		expectedError  string
```